### DISCUSSION TITLE
docker-maven-plugin and docker events

### DISCUSSION THREAD

**Discussion by ptoman-cisco** [fabric8io/docker-maven-plugin#1740] [Q&A] [answered]
Hello all,
We're using `docker events` (using https://www.jetbrains.com/help/teamcity/docker-support.html) to capture all events occurring  in our TeamCity CI/CD builds (for audit, monitoring, and other purposes).
Unfortunately it seems that docker pushes performed using the docker-maven-plugin are *not* captured by `docker events` at all for some reason.
I haven't gone through the sources yet; does anyone have any insight into this please?
Best regards 
Petr

**Comment by rohanKanojia** [comment]
Are you using `buildx` for performing docker push? For buildx functionality plugin relies on CLI rather than REST api

**Reply by ptoman-cisco** [reply, marked as answer]
@rohanKanojia no, we're using `docker-maven-plugin` both to build and to push the image.

Anyway, looking again into the TeamCity logs I can see that the event is being fired but it's not captured properly. It seems that the additional attributes in `Actor.Attributes` could be to blame; perhaps the TeamCity Docker plugin parser is confused about these. I'll raise this with the vendor (JetBrains).

Apologies for the false alarm, and thank you for the quick response.
